src/tags/if_block.rs

```diff
@@ -25,6 +26,19 @@ struct Conditional {
     if_false: Option<Template>,
 }
 
+fn contains_check(a: Value, b: Value) -> Result<bool> {
+    let a = match a.as_str() {
+        Some(s) => s,
+        None => return Error::renderer("Left-hand side of contains operator must be a string"),
+    };
+    let b = match b.as_str() {
+        Some(s) => s,
+        None => return Error::renderer("Right-hand side of contains operator must be a string"),
+    };
+
+    Ok(a.contains(b))
```

Comment:
`contains` supports two use cases
- substring matches (`page.title contains "Hello"`)
- checking a hashmap if it contains a key (`page contains "title"`)

This implements the first but for https://github.com/cobalt-org/cobalt.rs/issues/363, it sounds like you need the second.

If you want, I can merge this now.  Hopefully soon I can add support for checking for the object containing a key.  Otherwise, feel free to add support for that in this PR or another one.  Just let me know what you want.